backend/tests/unit/test_extract.py

```diff
@@ -3,37 +3,26 @@
 import pandas as pd
 import pytest
 
-from modules.common.domain.exceptions import (
-    DataValidationException,
-    FileDataFormatNotSupportedException,
-)
 from modules.extract.domain import commands as domain_commands
+from modules.extract.domain.exceptions import (
```

Comment:
We can't this file. Please see example of tests which I've sent.
We need to have test structure that matches modules folder.